alternator/executor.cc

```diff
@@ -2616,7 +2622,9 @@ future<executor::request_return_type> executor::put_item(client_state& client_st
 
     co_await verify_permission(_enforce_authorization, client_state, op->schema(), auth::permission::MODIFY);
 
-    if (auto shard = op->shard_for_execute(needs_read_before_write); shard) {
+    auto handle = op->get_erm_handle();
```

Comment:
Please consider a slightly less general name then "handle" (I am also not a big fan of "erm_handle" - I have no idea what it means).
If the purpose of the handle to avoid tablet migration, maybe we should call it tablet_migration_lock or something like that?

Reply:
I explained how it works above, I don't have a better explanation.

I'm fine with the name `erm_handle`. If you don't have a better alternative, then we should merge this one, or wait for suggestions from other reviewers.